docs/_articles/en/einstein/einstein-troubleshoot.md

```diff
@@ -26,12 +26,10 @@ Use this troubleshooting checklist to identify and eliminate common problems wit
 2. Check that your Einstein for Developers extension is activated using the **Developer: Show Running Extensions** command. If your extension isn’t activating, file an [issue](https://github.com/forcedotcom/Einstein-GPT-for-Developers/issues) in our GitHub repo.
 3. Make sure you’re connected to a Salesforce org. The status of your default org connection is visible in the status bar.
 
-
 ![Default Org](./images/default-org.png)
 
-
 **Tip**: Run the **SFDX: Open Default Org** command to easily check that your Salesforce extensions are running correctly and that you're connected to a Salesforce org.
 
+### Code Builder Environment Checklist
 
-
-
+1. Einstein for Developers extension is disabled in Code Builder by default. To enable the extension you must first set the Salesforce telemetry level to `true` for your VS Code workspace. Go to **Settings** and search for `salesforcedx-vscode-core.telemetry.enabled` in the **Workspace** tab to confirm or update. Telemetry can be disabled once the extension is enabled.
```

Comment:
Let's include whatever we do for the explanation above, in this section as well?